crates/locutus-node/src/node/test_utils.rs

```diff
@@ -57,8 +58,8 @@ pub(crate) type EventId = usize;
 
 #[derive(Clone)]
 pub(crate) struct NodeSpecification {
-    /// Pair of contract and the initial value
-    pub owned_contracts: Vec<(Contract, ContractValue)>,
+    /// Pair of contract, the initial value and contract subscribers
+    pub owned_contracts: Vec<(Contract, ContractValue, Vec<PeerKeyLocation>)>,
```

Comment:
I think it's better for clarity if we split this, leave owned_contracts as it is, and then add a new field which is a map of `ContractKey <- Vec<PeerKeyLocation>`.  You can relate the subscribers to the contracts b y the contract key later.